modules/cnf-understanding-low-latency.adoc

```diff
@@ -21,6 +21,11 @@ Administrators must be able to manage their many Edge sites and local services i
 
 {product-title} uses the Node Tuning Operator to implement automatic tuning to achieve low latency performance for {product-title} applications. The cluster administrator uses this performance profile configuration that makes it easier to make these changes in a more reliable way. The administrator can specify whether to update the kernel to kernel-rt, reserve CPUs for cluster and operating system housekeeping duties, including pod infra containers, and isolate CPUs for application containers to run the workloads.
 
+[NOTE]
+====
+Currently, disabling CPU load balancing is not supported by cgroup v2. As a result, you might not get the desired behavior from performance profiles if you have cgroup v2 enabled. It is recommended to not enable cgroup v2 if you are using performace profiles.
```

Comment:
we should probably have a consistent naming scheme for cgroupv2. above there is not a space and here there is. idc which is it really. kernel docs seem to use a space https://docs.kernel.org/admin-guide/cgroup-v2.html

Reply:
We use a space with cgroup v1 and cgroup v2. Thanks for catching the outlier!